plugins/catalog-import/src/components/ImportInfoCard/ImportInfoCard.tsx

```diff
@@ -46,8 +46,7 @@ export const ImportInfoCard = (props: ImportInfoCardProps) => {
   const appTitle = configApi.getOptional('app.title') || 'Backstage';
   const catalogImportApi = useApi(catalogImportApiRef);
 
-  const integrations = configApi.getConfig('integrations');
-  const hasGithubIntegration = integrations.has('github');
+  const hasGithubIntegration = configApi.has('integrations.github');
```

Comment:
@Rugvip I wonder though whether this could be dropped entirely.

A default integration will be added for github (without any credentials), but this will not be visible/obvious here as it only checks the config.

Additionally, `auth.providers.github` is used at the frontend when using the "github-only" feature here.
Of course, when you want to refresh it later, you will need to integration though.

But there is no check that there are credentials. Adding the default/public github integration explicitly would be sufficient for the check, too.

Reply:
ok, makes sense. Then I guess this change still makes sense as it is.